### DISCUSSION TITLE
Use Apache Tika as reader

### DISCUSSION THREAD

**Discussion by mykolamelnykml** [StabRise/spark-pdf#8] [Ideas]
Interesting to try Apache Tika as reader.

**Comment by muncleben** [comment]
Like:
df = spark.**readStream**.format("pdf") \
    .option("imageType", "BINARY") \
    .option("resolution", "200") \
    .option("pagePerPartition", "2") \
    .option("reader", "pdfBox") \
    .option("ocrConfig", "psm=11") \
    .option("password", "pdf_password") \
    .load("path to the pdf file(s)")

**Reply by mykolamelnykml** [reply]
@muncleben Do you want to have support structured streaming?
